tasks/clean/remove_domain_process.yml

```diff
@@ -33,8 +33,8 @@
     - name: Remove storage domain with no force
       include_tasks: tasks/clean/remove_domain.yml
       vars:
-          host: "{{ ovirt_hosts[0].id }}"
-      when: "ovirt_hosts is defined and ovirt_hosts|length > 0 and not dr_force"
```

Comment:
This should be changed, the facts module this returns `ovirt_hosts` fact.

Reply:
restored to ovirt_hosts